Tests/LibWeb/Text/input/HTML/submit-button-click-when-disabled.html

```diff
@@ -0,0 +1,12 @@
+<form id="theForm" style="display:none"><button id="theButton" type="submit" disabled></button></form>
+<script src="../include.js"></script>
+<script>
+    theForm.onsubmit = function() {
```

Comment:
This test passes when I run it on `master`.  Using `onclick` here instead causes the test to fail on `master`.

Reply:
Very observant!!